### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Wahaj-Ali**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
...
\ No newline at end of file
+# GitHub-Flow
+> In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+# Built With 
+-HTML
+-CSS
+🤵‍♂️Author:
+-Wahaj Ali
+# Show your support
+Give a ⭐ if you like this project.
```
Comment (fherrerao) [inline README.md]: - [ ] Good job adding, the Readme file, but I think you can improve it by editing/using the [Microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md), adding more information related to the project's information.

**Review by fherrerao — CHANGES_REQUESTED** [review]
Hi @Wahaj-Ali , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

So Close Snl GIF by Saturday Night Live


## Highlights

✅ Good commit messages.
✅ Good job adding `.html` and `.css` files.


## Required Changes ♻️
- [ ] Kindly, don't forget to add a descriptive title and summary 🛩️ telling what changes are inside.
Screenshot from 2022-10-25 08-23-09

- [ ] Good job adding linter files 💯 , some of them passed the test, but you forgot to check the linters for the styles, please fix it! 🦖.

- [ ] Excellent job adding a branch to use GitHub flow 😸, but I recommend using the correct use of branch names like `feature/hello-microverse` , `feature/set-up`, or similars 🦕, please change your branch name.


_Also, check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Wahaj-Ali** [comment]
Hi!
I am a bit confused because I cannot edit my files without opening a new pull request. I edited "readme.md" and "style.css". But as I did so, it asked me to open a new pull request.
If you can guide me through this process, I shall be very thankful.

Regards
    On Tuesday, October 25, 2022 at 06:47:40 PM GMT+5, Fernando Herrera ***@***.***> wrote:  
 
 


@fherrerao requested changes on this pull request.

Hi @Wahaj-Ali ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



Highlights

 ✅ Good commit messages.
 ✅ Good job adding .html and .css files.

Required Changes ♻️
    
   -    
 Kindly, don't forget to add a descriptive title and summary 🛩️ telling what changes are inside.   


   -    
 Good job adding linter files 💯 , some of them passed the test, but you forgot to check the linters for the styles, please fix it! 🦖 .

   -    
 Excellent job adding a branch to use GitHub flow 😸 , but I recommend using the correct use of branch names like feature/hello-microverse , feature/set-up, or similars 🦕 , please change your branch name.


Also, check the comments under the review.

Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

In README.md:
> -# GitHub-Flow
\ No newline at end of file
+# GitHub-Flow
+> In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+# Built With 
+-HTML
+-CSS
+🤵‍♂️Author:
+-Wahaj Ali
+# Show your support
+Give a ⭐ if you like this project.
   
   -  Good job adding, the Readme file, but I think you can improve it by editing/using the Microverse template, adding more information related to the project's information.

**Comment by Wahaj-Ali** [comment]
Hi!
Sorry for the inconvenience. I was confused about what to do and I took the wrong steps. I mistakenly closed the previous pull request that was to be re-reviewed. I am providing the link to a new pull request for review. 
Regards    On Tuesday, October 25, 2022 at 09:53:46 PM GMT+5, Wahaj Ali ***@***.***> wrote:  
 
  Hi!
I am a bit confused because I cannot edit my files without opening a new pull request. I edited "readme.md" and "style.css". But as I did so, it asked me to open a new pull request.
If you can guide me through this process, I shall be very thankful.

Regards
    On Tuesday, October 25, 2022 at 06:47:40 PM GMT+5, Fernando Herrera ***@***.***> wrote:  
 
 


@fherrerao requested changes on this pull request.

Hi @Wahaj-Ali ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



Highlights

 ✅ Good commit messages.
 ✅ Good job adding .html and .css files.

Required Changes ♻️
    
   -    
 Kindly, don't forget to add a descriptive title and summary 🛩️ telling what changes are inside.   


   -    
 Good job adding linter files 💯 , some of them passed the test, but you forgot to check the linters for the styles, please fix it! 🦖 .

   -    
 Excellent job adding a branch to use GitHub flow 😸 , but I recommend using the correct use of branch names like feature/hello-microverse , feature/set-up, or similars 🦕 , please change your branch name.


Also, check the comments under the review.

Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

In README.md:
> -# GitHub-Flow
\ No newline at end of file
+# GitHub-Flow
+> In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+# Built With 
+-HTML
+-CSS
+🤵‍♂️Author:
+-Wahaj Ali
+# Show your support
+Give a ⭐ if you like this project.
   
   -  Good job adding, the Readme file, but I think you can improve it by editing/using the Microverse template, adding more information related to the project's information.

**Pull request closed without merging**